src/chrome/content/rules/PoliceUK.xml

```diff
@@ -31,5 +32,8 @@
 	<target host="police.uk" />
 	<target host="www.police.uk" />
 
-  <rule from="^http://(www\.)?police\.uk/" to="https://www.police.uk/" />
+	<test url="http://police.uk/" />
```

Comment:
You can delete this since there is an implicit test when you add the target URL.